File: clients/csharp/Bots/RandomAi.cs
using CsClient.Bots.Internal;

namespace CsClient.Bots
{
    public class RandomAi : IBot
    {
        /// <inheritdoc/>
        public string Name { get; }

        /// <inheritdoc/>
        public int Play(PlayState? playState)
        {
            Random random = new Random();
            return random.Next(0, 7);
        }

        public RandomAi()
        {
            Name = "RandomAiCSharp";
        }
    }
}

File: clients/python/Bots/random_ai.py
from random import randrange

from Bots.bot_ai import BotAI
from Bots.data import PlayState


class RandomAI(BotAI):
    column = 0

    def play(self, state: PlayState):
        col = randrange(0, 7)
        return col

    def _print_board(self, board):
        for row in board[::-1]:
            print(row)
        print("")

    def get_name(self):
        return self.name

    def __init__(self):
        self.name = "RandomAI"
